website/docs/r/pod_security_policy.html.markdown

```diff
@@ -0,0 +1,179 @@
+---
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_pod_security_policy"
+sidebar_current: "docs-kubernetes-resource-pod-security-policy"
+description: |-
+  A Pod Security Policy is a cluster-level resource that controls security sensitive aspects of the pod specification.
+---
+
+# kubernetes_api_service
```

Comment:
This should say `kubernetes_pod_security_policy`

Reply:
Fixed